research/00-introduction.tex

```diff
@@ -2,6 +2,31 @@ \section*{Introduction}
 \label{sec:introduction}
 \addcontentsline{toc}{section}{Introduction}
 
+Spoofax is a language workbench developed by Delft University of Technology over
+the course of several years. During those years it has grown to \textit{"a
+platform for developing textual domain-specific languages with full-featured
+Eclipse editor plugins"}~\cite{SpoofaxWebsite}. Today languages in Spoofax are
+constructed using the meta-languages `SDF3', `NABL', `TS' and `Stratego', where
+each meta-language has a specific purpose. These meta-languages in turn are used
+to generate several language services, such as syntax highlighting, code
+transformation and error marking.
```

Comment:
That's a good point. Perhaps we should cite section 1 :laughing: 

I think we can remove the last part:
> Today languages in Spoofax are constructed using the meta-languages \`SDF3', \`NABL', \`TS' and \`Stratego', where each meta-language has a specific purpose. These meta-languages in turn are used to generate several language services, such as syntax highlighting, code transformation and error marking.